components/apply-link/index.js

```diff
@@ -52,12 +53,17 @@ const StyledLink = styled.a`
   }
 `
 
-const ApplyLink = (props) => (
+const ApplyLink = ({displayedText, ...props}) => (
   <Link href="/apply" passHref>
     <StyledLink rel="noopener" {...props}>
-      <span>Apply to Join Us</span>
+      <span>{displayedText}</span>
```

Comment:
Is there a reason to make the ApplyLink dynamic? I'm assuming the text stays static because it's reused in a few places? Do you want to make another generic Link component to support dynamic text?

Reply:
Combined files with the button component. Button component is used in the submit form when applying. Created LinkButton, and added an ApplyLink which is a button that uses LinkButton but adds href and text. Named it LinkButton instead of Link to remove ambiguity with the Next.js Link